packages/datetime/src/_daterangepicker.scss

```diff
@@ -63,20 +63,45 @@ Styleguide components.datetime.daterangepicker.js
     margin-right: $pt-grid-size;
   }
 
+  .DayPicker-Day--hovered-range:not(.DayPicker-Day--outside) {
+    border-radius: 0;
+
+    // need to disable hover-styles for all variants of selected dates
+    // stylelint-disable max-line-length
+    &:not(.DayPicker-Day--selected):not(.DayPicker-Day--selected-range):not(.DayPicker-Day--selected-range-start):not(.DayPicker-Day--selected-range-end) {
```

Comment:
yuck! why do you have to do this? 
i wonder if we can find another way (though perhaps in a separate PR).
can we just define this one first so the others will override by virtue of coming later?